Description: Disable libunwind integration
Bug-Debian: https://bugs.debian.org/1093688
Forwarded: no
Author: Dylan Aïssi <daissi@debian.org>

--- a/meson_options.txt
+++ b/meson_options.txt
@@ -42,6 +42,11 @@ option('lc-compliance',
         value : 'auto',
         description : 'Compile the lc-compliance test application')
 
+option('libunwind',
+        type : 'feature',
+        value : 'auto',
+        description : 'Enable libunwind provide backtraces to help debugging')
+
 option('pipelines',
         type : 'array',
         value : ['auto'],
--- a/src/libcamera/base/meson.build
+++ b/src/libcamera/base/meson.build
@@ -27,7 +27,7 @@ libcamera_base_internal_sources = files(
 ])
 
 libdw = dependency('libdw', required : false)
-libunwind = dependency('libunwind', required : false)
+libunwind = dependency('libunwind', required : get_option('libunwind'))
 
 if cc.has_header_symbol('execinfo.h', 'backtrace')
     config_h.set('HAVE_BACKTRACE', 1)
